float64[] cont
int64 contDim

int64 disc



